File: services/user-portal/src/main/java/com/researchex/userportal/api/ServiceMetadataResponse.java
package com.researchex.userportal.api;

import java.time.OffsetDateTime;

/**
 * User Portal 서비스가 반환하는 메타데이터 응답 모델.
 */
public record ServiceMetadataResponse(
        String serviceName, String description, OffsetDateTime timestamp) {}

File: services/user-portal/src/main/kotlin/com/researchex/userportal/api/ServiceMetadataResponse.kt
package com.researchex.userportal.api

import java.time.OffsetDateTime

/** User Portal 서비스가 반환하는 메타데이터 응답 모델이다. */
data class ServiceMetadataResponse(
    val serviceName: String,
    val description: String,
    val timestamp: OffsetDateTime,
)
